java/src/test/java/ai/rapids/cudf/ReductionTest.java

```diff
@@ -399,27 +399,31 @@ private static Stream<Arguments> createFloatArrayParams() {
   }
 
   private static void assertEqualsDelta(ReductionAggregation op, Scalar expected, Scalar result,
-      Double percentage) {
+                                        Double percentage) {
     if (FLOAT_REDUCTIONS.contains(op.getWrapped().kind)) {
       assertEqualsWithinPercentage(expected.getDouble(), result.getDouble(), percentage);
     } else if (expected.getType().typeId == DType.DTypeEnum.LIST) {
-      try (ColumnView e = expected.getListAsColumnView();
-           ColumnView r = result.getListAsColumnView()) {
-        AssertUtils.assertColumnsAreEqual(e, r);
+      try (ColumnVector expectedAsList = ColumnVector.fromScalar(expected, 1);
+           ColumnVector resultAsList = ColumnVector.fromScalar(result, 1);
```

Comment:
Similar "why not just use the column view and avoid the copy-to-column" comment here.